PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/explorer/ResourceConnectorTree.java

```diff
@@ -32,12 +35,20 @@
 import java.util.Arrays;
 import java.util.Comparator;
 import java.util.Enumeration;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 
 public class ResourceConnectorTree extends AzureTree {
 
     private final Project project;
+    private static final Map<Class<? extends Resource>, Icon> RESOURCE_ICON_MAP = new HashMap<>();
+
+    static {
+        RESOURCE_ICON_MAP.put(MySQLDatabaseResource.class, AzureIconLoader.loadIcon(AzureIconSymbol.MySQL.MODULE));
```

Comment:
because this map is used internal only, use resourceIconMap is fine, just like `private static final Logger logger =`

Reply:
renamed